src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -11,7 +11,7 @@
 public class CommandResult {
 
     /** The feedback message to be shown to the user. Contains a description of the execution result */
-    public final String feedbackToUser;
+    private final String feedbackToUser;
```

Comment:
these are changes from T3A2. do remember what we discussed in class on how to ensure proper branching techniques can help here